test/testutils/syncenginetestutils.cpp

```diff
@@ -1106,9 +1090,10 @@ void FakeFolder::fromDisk(QDir &dir, FileInfo &templateFi)
             fi.setLastModified(diskChild.lastModified());
             if (fi.isDehydratedPlaceholder) {
                 fi.contentChar = '\0';
+                fi.contentSize = 0;
             } else {
                 QFile f { diskChild.filePath() };
-                f.open(QFile::ReadOnly);
+                OC_ENFORCE(f.open(QFile::ReadOnly));
```

Comment:
A regular assert should do. There is no point in building unit tests in release mode anyway, all our asserts in the business code would never run.

Reply:
We're doing a release build, and asserts are off.